src/components/Common/Navigation/BreadcrumbWrap/BreadcrumbWrap.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * This function is used to render the breadcrumb for the Agent Issuer, Agent
+ * Holder, and Agent Verifier pages
```

Comment:
Same as the other repo

Reply:
this is why I don't like comments like that in the code, if code and structure does not speak for itself, then README should back it up. Thanks for highlighting, going to update this repo shortly.